filament/src/materials/ssao/sao.mat

```diff
@@ -216,7 +216,7 @@ fragment {
             tapPosition = angleStep * tapPosition;
         }
 
-        float ao = max(0.0, 1.0 - occlusion * materialParams.intensity);
+        float ao = max(0.0, 1.0 - sqrt(occlusion * materialParams.intensity));
```

Comment:
Is the extra sqrt necessary?

Reply:
yeah, it helps a lot making the effect smoother and more consistant at all quality levels -- it's somewhat an artistic decision though. Before we achieved a similar effect by increasing the contrast, but it didn't look as  nice.